src/components/Disputes/DisputeTimeline.js

```diff
@@ -302,4 +292,8 @@ const StyledAccordion = styled.div`
   }
 `
 
-export default DisputeTimeline
+export default function DisputeTimeline({ dispute }) {
+  const disputeTimeLine = useDisputeTimeline(dispute)
```

Comment:
Not sure if the `memo()` is really that beneficial (since the timeline object would change on each hook re-run anyway?), otherwise we could consider inlining this hook with `Timeline`?

Reply:
Are you referring to the memoized component or the memoized timeline value?

The problem with inlining the hook is that the `dispute` object is a new one on every poll which will cause a re-render on `Timeline` and causing the auto-collapsing issue.

Reply:
Hmm unless I'm misunderstanding something, I'm not sure how this fixes the issue.

If `disputeTimeline` is a new value on every poll, then `Timeline`'s `timeline` prop will be different on every poll, and even with the `memo()` applied, we'd still be re-rendering `Timeline`?

Reply:
Ah no, the new value on every poll is `dispute`, `useDisputeTimeline` memoizes the `disputeTimeline` value with [these values](https://github.com/aragon/court-dashboard/blob/f040a43ff1d91f92d86699eddaf907a9085cce3a/src/hooks/useDisputeTimeline.js#L96) as dependencies.

Reply:
Got it, that finally makes sense for me 😄. Thanks for the explanation 🙇‍♂️!

Reply:
No problem! :)